chpl/chpl-service/src/main/java/gov/healthit/chpl/manager/auth/UserManager.java

```diff
@@ -189,9 +194,14 @@ public void updateFailedLoginCount(UserDTO userToUpdate) throws UserRetrievalExc
         String maxLoginsStr = env.getProperty("authMaximumLoginAttempts");
         int maxLogins = Integer.parseInt(maxLoginsStr);
 
-        if (userToUpdate.getFailedLoginCount() >= maxLogins) {
+        if (userToUpdate.getFailedLoginCount() == maxLogins) {
```

Comment:
I know it's unlikely, but if something happened where the failed login count jumped over the maxLogins value, and somehow didn't actually lock the account, then we never would, right? Can we keep this comparison as `>=` and add a separate `if` inside this block to send the email only when the `==` comparison is true?